packages/manager/modules/user-contracts/src/index.js

```diff
@@ -0,0 +1,34 @@
+import angular from 'angular';
+
+import { includes, isEmpty, isArray } from 'lodash';
+import '@ovh-ux/manager-core';
+import '@ovh-ux/ui-kit';
+import '@uirouter/angularjs';
+import 'angular-translate';
+
+import component from './user-contracts.component';
+import service from './user-contracts.service';
+import { CONTRACTS_LIST } from './user-contracts.constant';
+
+const moduleName = 'ovhManagerUserContracts';
+
+angular
+  .module(moduleName, ['ovhManagerCore', 'pascalprecht.translate', 'ui.router'])
+  .component('userContracts', component)
+  .service('userContractsService', service)
+  .run(/* @ngTranslationsInject:json ./translations */)
+  .run(($q, $state, coreConfig, userContractsService) =>
+    userContractsService
+      .getAgreementsToValidate((contract) =>
+        includes(CONTRACTS_LIST, contract.code),
+      )
+      .then((contracts) => {
+        const region = coreConfig.getRegion();
+        if (region === 'US' && isArray(contracts) && !isEmpty(contracts)) {
+          $state.go('app.userContracts', { contracts });
```

Comment:
I think this should be done in the app using the module. Else, it makes this module dependent on the app (the app must have a `app.userContracts` state)

Reply:
Lot of code will be repeated if i try to make it generic. Can we leave this specific to US region only and for TOS and PP ?
I dont see this implemented for any other regions.

Reply:
ok... May be we can have it that way for now...